bundles/org.openhab.binding.mybmw/README.md

```diff
@@ -145,6 +146,21 @@ They differ for each vehicle type, build-in sensors and activated services.
 | [tires](#tire-pressure)          | Current and wanted pressure for all tires         |  X   |  X   |    X    |  X  |
 | [image](#image)                  | Provides an image of your vehicle                 |  X   |  X   |    X    |  X  |
 
+#### Vehicle Update
+
+Provides switches to force partly updates of the BMW API functionality
+
+- Channel Group ID is status
+- Available for all vehicles
+- switches which can be triggered by a command
+- if the switches are set to ON, then immediately they will be set to OFF again for being able to trigger the next update
+
+| Channel Label                                  | Channel ID      | Type   | Description                                                               | conv | phev | bev_rex | bev |
+|------------------------------------------------|-----------------|--------|---------------------------------------------------------------------------|------|------|---------|-----|
+| trigger the update of the state                | state-update    | Switch | When set to ON, the state of the vehicle will be updated                  |  X   |  X   |    X    |  X  |
+| trigger the update of the charging information | charging-update | Switch | When set to ON, the charging statistics of the vehicle will be updated    |      |  X   |    X    |  X  |
+| trigger the update of the image                | image-update    | Switch | When set to ON, the image of the vehicle will be updated                  |  X   |  X   |    X    |  X  |
```

Comment:
I would suggest to drop the Channel Label column, that can easily be conficting with translations and the Channel Id is allready more accurate.

Reply:
was thinking about the same, just did it due to continuity like in the other chapters. Removed all of them